setup.cfg

```diff
@@ -2,3 +2,4 @@
 addopts = -n auto
 markers =
     large: marks tests as large (deselect with '-m "not large"')
+    deprecated: marks tests that cover deprecated code
```

Comment:
in what sense do they cover deprecated code? can you explain this more. deprecated code still needs to be tested.

what code is deprecated? are the tests deprecated or the code they test?

Reply:
This is used to mark cases in which a V2 policy is running in a V1 environment. We could be very specific and mark it as `V2_policy_in_V1_env`, but that would only get used once in the codebase. I figured something like `deprecated` could potentially be used elsewhere as well — for example, on the test in which we instantiate every environment (that test is implicitly covered in the scripted policy tests). Maybe we could rename it to `extra_test`? I’m open to suggestions.

Reply:
i just don't understand what's deprecated about anything here? as far as i can tell, all of this code is still in use and will be for some time.

i think `extra` is a fine mark if you just want some tests which most people aren't interested in.